index.html

```diff
@@ -18,7 +18,8 @@ <h3>So much coolness</h3>
         <section>
             <h3>Also here's some more content</h3>
             <p>Such content. Very wow.</p>
-            <img src="https://media1.tenor.com/images/a53e378bb32fdbcabd28565ae799cfd5/tenor.gif?itemid=15912640" alt="">
+            <img src="https://media1.tenor.com/images/a53e378bb32fdbcabd28565ae799cfd5/tenor.gif?itemid=15912640" alt="Baby Yoda wave" />
+            <img src="https://media.giphy.com/media/KziKCpvrGngHbYjaUF/giphy.gif" alt="Baby Yoda sip" />
```

Comment:
Great img source, Owen! Could you please update the alt tag to be more descriptive.

Reply:
Done and done! Thanks for the feedback, Cap'n!